### DISCUSSION TITLE
短信告警应当开放给所有用户使用

### DISCUSSION THREAD

**Discussion by Xeath** [1Panel-dev/1Panel#6369] [General]
<img width="1484" alt="截屏2024-09-05 10 43 48" src="https://github.com/user-attachments/assets/1d96bc89-b861-409e-82c0-f33d81b5382b">

实在是想不明白这个功能为什么只限制专业版用户使用，希望官方人员可以进行解答。

而我的理解就是，这个功能应该开放给包含免费用户的所有 1Panel 用户使用，原因也很简单：

1. 目前仅提供给专业版用户使用，专业版用户就不需要购买短信包了吗？如果 1Panel 目前的政策就是 1Panel Pro 一次购买短信永久免费使用那我无话可说；
2. 提供给免费版本用户使用，但是绝对不是免费提供，而是需要购买短信包（很正常，1Panel 使用第三方接口也要被收费），这可以让 1Panel 在 Pro 之外多了一个营收的方式，这对 1Panel 用户还是官方只有好处没有坏处。

另外我看到有 Issue 提出的问题 [#614](https://github.com/1Panel-dev/1Panel/issues/614)，1Panel 是否有计划开发自定义通知功能？该项功能隔壁友商是免费使用（除短信）。

<img width="1057" alt="截屏2024-09-05 10 53 31" src="https://github.com/user-attachments/assets/fcde0081-51df-4d44-a25c-c57845ffd34e">

告警功能是面板的一项基础功能，国外面板大多支持 Email 告警，没必要把一项非常基础的功能作为付费功能。

比如 1Panel 多节点管理等高级功能收费我觉得情有可原，甚至围绕 Docker 开发的高级功能收费我都觉得非常合理。

如果说是涉及到账户绑定问题才做成付费功能，我到觉得可以考虑和第三方平台一样，做成 ID + Key 的形式，要求用户去 1Panel 官方注册账户并购买短信包，然后提供 ID 和 Key 让用户绑定的面板，这样就可以绕过账户绑定的问题。

当然我也知道，做成可以自定义的功能之后，很多人都会自己对接短信接口或者 App 通知（比如 PushDeer、Bark 等）。

**Comment by wanghe-fit2cloud** [comment]
https://github.com/1Panel-dev/1Panel/issues/614
